src/a-d/Aroon/Aroon.BufferList.cs

```diff
@@ -16,6 +16,8 @@ public AroonList(int lookbackPeriods = 25)
         Aroon.Validate(lookbackPeriods);
         LookbackPeriods = lookbackPeriods;
         _buffer = new Queue<(DateTime, double, double)>(lookbackPeriods + 1);
+
+        Name = $"AROON({25})";
```

Comment:
Is this right?  Looks like you're hardcoding the default, not the input param value.
Potentially many places.

```suggestion
        Name = $"AROON({lookbackPeriods})";
```